docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -0,0 +1,496 @@
+---
+title: What's new in .NET Core 3.0
+description: Learn about the new features found in .NET Core 3.0.
+dev_langs: 
+  - "csharp"
+  - "vb"
+author: thraka
+ms.author: adegeo
+ms.date: 12/04/2018
+---
+
+# What's new in .NET Core 3.0 (Preview 1)
+
+This article describes what is new in .NET Core 3.0 (preview 1). One of the biggest enhancements is support for Windows desktop applications (Windows only). By utilizing a .NET Core 3.0 component called Windows Desktop, you can port your Windows Forms Windows Presentation Foundation (WPF) applications. To be clear, the Windows Desktop component is only supported on Windows. For more information, see the section [Windows desktop](#windows-desktop) below.
+
+.NET Core 3.0 adds support for C# 8.0.
+
+[Download and get started with .NET Core 3 Preview 1](https://aka.ms/netcore3download) right now on Windows, Mac and Linux. You can see complete details of the release in the [.NET Core 3 Preview 1 release notes](https://aka.ms/netcore3releasenotes).
+
+For more information, see the [.NET Core 3.0 Preview 1 announcement](https://blogs.msdn.microsoft.com/dotnet/2018/12/04/announcing-net-core-3-preview-1-and-open-sourcing-windows-desktop-frameworks/).
+
+## .NET Standard 2.1
+
+.NET Core 3.0 will be compliant with .NET Standard 2.1.
```

Comment:
nit: would it be more correct to say ".NET Core 3.0 implements .NET Standard 2.1."?

Reply:
Thanks, I wasn't sure about the wording there.